pacman/executor/algorithm_classes/python_class_algorithm.py

```diff
@@ -61,10 +60,10 @@ def call_python(self, inputs):
             method = "__call__"
             if self._python_method is not None:
                 method = self._python_method
-            exc_type, exc_value, exc_trace = sys.exc_info()
-            logger.error("Error when calling {}.{}.{} with inputs {}".format(
-                self._python_module, self._python_class, method, inputs))
-            raise exc_type, exc_value, exc_trace
+            logger.error("Error when calling %s.%s.%s with inputs %s",
```

Comment:
this goes against python recommendations of using .format. So why does pylint demand it?